src/app/_components/Footer.tsx

```diff
@@ -1,16 +1,15 @@
-import { Button } from '@/components/Button'
 import { Logo } from '@/components/Logo'
+import { NewsletterForm } from '@/components/NewsletterForm'
 import { Social } from '@/components/Social'
 import { TextLink } from '@/components/TextLink'
 
 import { footerNavigationItems } from '@/constants/navigationItems'
-import { FILECOIN_FOUNDATION_URLS } from '@/constants/siteMetadata'
 
 export function Footer() {
   return (
     <footer className="mt-16 flex flex-col gap-6">
       <hr />
-      <div className="grid grid-cols-1 gap-y-6 sm:grid-cols-5">
+      <div className="my-2 grid grid-cols-1 gap-y-6 sm:grid-cols-5">
```

Comment:
I think we might need to increase this so there's also some space when there's an error message.